packages/client/src/containers/LandingPage/LandingPage.Container.js

```diff
@@ -1,25 +1,85 @@
-import React, { useEffect, useState } from 'react';
-import { apiURL } from '../../apiURL';
-import './LandingPage.Style.css';
+// import React, { useEffect, useState } from 'react';
+// import { apiURL } from '../../apiURL';
+// import './LandingPage.Style.css';
+
+// export const LandingPage = () => {
+//   const [exampleResources, setExampleResources] = useState([]);
+//   useEffect(() => {
+//     async function fetchExampleResources() {
+//       const response = await fetch(`${apiURL()}/exampleResources`);
+//       const examples = await response.json();
+//       setExampleResources(examples);
+//     }
 
-export const LandingPage = () => {
-  const [exampleResources, setExampleResources] = useState([]);
-  useEffect(() => {
-    async function fetchExampleResources() {
-      const response = await fetch(`${apiURL()}/exampleResources`);
-      const examples = await response.json();
-      setExampleResources(examples);
-    }
+//     fetchExampleResources();
+//   }, []);
 
-    fetchExampleResources();
-  }, []);
+//   return (
+//     <div className="landing-page-container">
+//       <span>Landing Page</span>
+//       {exampleResources.map((example) => (
+//         <div key={example.id}>{example.title}</div>
+//       ))}
+//     </div>
+//   );
+// };
+
+import React from 'react';
+import Testimonials from '../../components/Login/Testimonials';
+import logo from './logo.svg';
+import { Typography, TextField, Button } from '@mui/material';
+import styled from '@emotion/styled';
+import './LandingPage.Style.css';
+// Import necessary components from Material-UI
 
+const CustomButton = styled(Button)({
+  padding: '14px 20px',
+  backgroundColor: 'var(--primary-color)',
+  color: 'white',
+});
+
+const LandingPageContainer = () => {
   return (
-    <div className="landing-page-container">
-      <span>Landing Page</span>
-      {exampleResources.map((example) => (
-        <div key={example.id}>{example.title}</div>
-      ))}
+    <div className="login-page">
+      <div className="testimonial-container">
+        <Typography className="text-field" variant="h4" color="white">
+          Don’t forget to invite teammates!
+        </Typography>
+        <div className="testimonial-section">
+          <div className="testimonial-cards">
+            <Testimonials /> {/* Render the Testimonials component */}
+          </div>
+        </div>
+      </div>
+
+      <div className="login-container">
+        <img className="logo" src={logo} alt="Logo" />
+        <div className="login-details">
+          <Typography variant="h5" fontWeight="bold">
+            Log in to Teams Check-in
+          </Typography>
+          <div className="login-action">
+            <div className="login-text">
+              <Typography className="text-field" variant="body1">
+                Enter your team code
+              </Typography>
+              <TextField
```

Comment:
Next up is to create a function and a variable to catch what the user wrote :)
You can use a console.log to print out the user's input! But this can be a separate PR

Reply:
Sure! I will create a next PR to integrate the backend to catch the user input!